Add documentation

Myrddhim opened the pull request:
test1 test of the new git systeme

Myrddhim committed:
test1

test1 test of the new git systeme

Myrddhim committed:
test2

arase of the sys test changes

GISED-Link commented:
Yep, I can see it

GISED-Link commented:
As you can see, Merging requires my review. This may be disabled in a while ... I will let this as it is for now

Myrddhim committed:
ajout doc

Myrddhim committed:
etude PoE carte

regarder les composant de la carte leurs fonctionnalités et étudier des altenative (sur plusieurs versions de carte car la dernière version comporte de composant non-disponibles)

Myrddhim committed:
réorganisation du Git

Myrddhim committed:
CarteFilleBlockDiagram.drawio

Myrddhim committed:
CarteFilleBlockDiagram.drawio

Myrddhim committed:
Coose of PCA9673

as an interrupt output

Myrddhim committed:
Merge branch 'dev_local_ts' of https://github.com/Myrddhim/MQTT_Keypad into dev_local_ts

Myrddhim committed:
BlockDiagram_carte_fille.drawio.pdf

Myrddhim committed:
BlockdiagramCarteFilleDébut

début du choix de composant et du blockdiagram de la carte fille

Myrddhim committed:
Merge branch 'dev_local_ts' of https://github.com/Myrddhim/MQTT_Keypad into dev_local_ts

Myrddhim committed:
SoftwarePerifFunctionalAnalisis1.drawio

Myrddhim committed:
SoftCartePerifFunkDiag1.drawio
